README.md

```diff
@@ -5,9 +5,9 @@ Mailmerge
 [![Build Status](https://travis-ci.com/awdeorio/mailmerge.svg?branch=develop)](https://travis-ci.com/awdeorio/mailmerge)
 [![codecov](https://codecov.io/gh/awdeorio/mailmerge/branch/develop/graph/badge.svg)](https://codecov.io/gh/awdeorio/mailmerge)
 
-A simple, command line mail merge tool.
+A simple, command line [mail merge](https://en.wikipedia.org/wiki/Mail_merge) tool.
```

Comment:
I think I'd prefer to keep the original text on this one.

Reply:
Sure, if you say so... I had no idea "mail merge" was a standard term for this, so this looked like a poor choice of words until I googled it (which was after I read all of the README).

Reply:
Reverted to the original text, assuming that's what you meant with your thumbs up.